docs/en/stack/security/authorization/privileges.asciidoc

```diff
@@ -82,6 +92,10 @@ All `monitor` privileges plus index administration (aliases, analyze, cache clea
 close, delete, exists, flush, mapping, open, force merge, refresh, settings,
 search shards, templates, validate).
 
+`manage_follow_index`::
+All actions that are required to manage a follower index, which includes pausing
```

Comment:
s/pausing and resuming/creating follow index, closing the index and unfollow the index.

Not that for pause and resume start the follow tasks and then read changes from leader shards and write changes into follower shards. For this write level index privilege is needed in follower index and read privilege is needed in leader index. The `manage_ccr` cluster level privilege is needed to start and stop shard follow tasks as part of resume and stop respectively.

Reply:
I would highlight that this is an index privilege required in the following cluster only, giving the ability on the index level to manage the lifecycle of follow indices.

Reply:
@martijnvg Other X-Pack API reference pages have an "Authorization" section in them, so I've drafted that content in https://github.com/elastic/elasticsearch/pull/35557 based on your comments here.  If it's useful, I can add that section to the other CCR API pages too.

Reply:
👍 thanks!